sys/auto_nimble_advertise/Makefile

```diff
@@ -0,0 +1,5 @@
+ifeq (gnu, $(TOOLCHAIN))
+  CFLAGS += -Wno-cast-function-type
```

Comment:
maybe better fix the code instead of suppressing the warning?!

Reply:
I removed this one. Honestly, I don't even remember anymore the reason why I've added this. There is not even a warning after removing this.